app/views/examples/click_to_edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Click to edit</h1>
+
+<h4>This pattern provides inline editing without a page refresh</h4>
+
+<div hx-target='this' hx-swap='outerHTML'>
```

Comment:
How about doing a helper for this?

Reply:
This is also a feature we want to provide. I think the syntax for this `div` could be like:
```ruby
<%= content_tag(:div, data: { htmx: { target: 'this', swap: 'outerHTML' } }) do %>
  ...
<% end %>
```

WDYT?

Reply:
I like it, how about calling it `htmx_swap_outer`
```ruby
<%= htmx_swap_outer do %>
  ...
<% end %>
```